### PR TITLE
Feature/fmcacov

### PR DESCRIPTION
**Pull request by ghbm-itk**
### Added

- SAP.fmcacov module 
  - Function to dismiss "key-popup" in SAP.
  - Function to reliably open forretningspartners in SAP.
- Test for all of the above.

### Changed

- opret_kundekontakt uses the new SAP.fmcacov module.
- Updated Github actions dependencies.

### Removed

- SAP login using web portal. Wasn't used or maintained.

### CODE REVIEW

File: itk_dev_shared_components/sap/fmcacov.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+"""This module contains functions to help with actions in the fmcacov transaction."""
+
+
+def find_forretningspartner(session, fp: str) -> None:
```
Comment (jekuaitk) [inline itk_dev_shared_components/sap/fmcacov.py:4]: Should we rename this to 'open_forretningspartner'?

File: itk_dev_shared_components/sap/fmcacov.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                # Press 'Accept' button.
+                session.FindById('wnd[1]/tbar[0]/btn[0]').press()
+                break
+        else:
+            # Range exhausted
+            raise ValueError(f"Forretningspartner '{fp}' was not found in pop-up.")
+
+
+def dismiss_key_popup(session, fp: str = "25564617") -> None:
+    """Once a day a popup appears asking to generate af new "afstemningsnøgle".
```
Comment (jekuaitk) [inline itk_dev_shared_components/sap/fmcacov.py:37]:
```suggestion
    """Once a day a popup appears asking to generate a new "afstemningsnøgle".
```

File: itk_dev_shared_components/sap/fmcacov.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        session: The SAP session object.
+        fp: The forretningspartner to used. Defaults to "25564617" a fictional person in AAK.
+
+    Raises:
+        RuntimeError: If a popup other than the expected ones appear.
+    """
+    find_forretningspartner(session, fp)
+
+    # Right click the first row in the postliste and click "Kontovedligehold med filter"
+    postliste = session.findById("wnd[0]/usr/tabsDATA_DISP/tabpDATA_DISP_FC1/ssubDATA_DISP_SCA:RFMCA_COV:0202/cntlRFMCA_COV_0100_CONT5/shellcont/shell")
```
Comment (jekuaitk) [inline itk_dev_shared_components/sap/fmcacov.py:52]: 🤯

Comment (ghbm-itk) [reply itk_dev_shared_components/sap/fmcacov.py:52]: SAP selectors 😎

**Review by jekuaitk — APPROVED** [review]

**Pull request merged**